tools/ffmpeg/ffmpeg.go

```diff
@@ -15,12 +15,12 @@ import (
 func main() {
 	targetDir, _ := filepath.Abs(os.Args[1])
 
-	archiveName := "ffmpeg-n5.1-latest-linux64-gpl-shared-5.1"
+	archiveName := "ffmpeg-n6.1-latest-linux64-gpl-shared-6.1"
 	if runtime.GOOS == "windows" {
-		archiveName = "ffmpeg-n5.1-latest-win64-gpl-shared-5.1"
+		archiveName = "ffmpeg-n6.1-latest-win64-gpl-shared-6.1"
 	}
 
-	downloadUrl := "https://github.com/Wieku/FFmpeg-Builds/releases/download/latest/" + archiveName + ".zip"
+	downloadUrl := "https://github.com/BtbN/FFmpeg-Builds/releases/download/latest/" + archiveName + ".zip"
```

Comment:
Please don't relink to BtBN builds, there's a reason why custom builds are used (smaller size as we don't need all encoders/decoders). 

Most recent [build](https://github.com/Wieku/FFmpeg-Builds/releases/tag/latest) should have av1_nvenc support added.

Reply:
ah i see, will get to it later (I linked to it because the custom builds didnt have av1 before)